docs/Reference/API-Methods.md

```diff
@@ -4390,6 +4390,116 @@ None
     methods, use the [`--rpc-http-api`](CLI/CLI-Syntax.md#rpc-http-api) or
     [`--rpc-ws-api`](CLI/CLI-Syntax.md#rpc-ws-api) options.
 
+### txpool_besuPendingTransactions
+
+Lists pending transactions that match the supplied filter conditions.
+
+
+#### Parameters
+
+* `QUANTITY` - Integer representing the maximum number of results to return.
+* Object of fields used to create the filter condition.
+
+Each field in the object corresponds to a field name containing an operator, and a value for the
+operator. The same field name can be specified multiple times, but can only contain one operator.
```

Comment:
I think this makes sense but I think it would be good to include an example where the same field name is specified multiple times.  ie, From reading this I assume you can have a request with params such as: 

 "params":[
          2,
          {
             "from":{
                "eq":"0xfe3b557e8fb62b89f4916b721be55ceb828dbd73",
                "eq":"0x627306090abaB3A6e1400e9345bC60c78a8BEf57"
             },
             "gas":{
                "lt":"0x5209"
             },
             "nonce":{
                "gt":"0x1"
             }
          }
       ],

Reply:
Also, this won't work.  We will only get one of the EQ fields.  if we tried GT and LT in the same operator it will fail because we explicitly check for it.